eslint.config.mjs

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+import { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export default tseslint.config({
+  files: [
+    'src/**/*.ts',
+    'spec/**/*.ts',
+  ],
+  extends: [
+    eslint.configs.recommended,
+    ...tseslint.configs.recommendedTypeChecked,
+    {
+      languageOptions: {
+        parserOptions: {
+          projectService: true,
+          tsconfigRootDir: __dirname,
+        },
+      },
+    },
+  ],
+  rules: {
+    'no-prototype-builtins': 'off',
+    'prefer-const': 'off', // TODO: enable rule and run autofix
```

Comment:
I don't think this TODO should go into main... can you remove it now, did you run autofix?

Reply:
There are a ton of changes that this would produce - so I'd rather do it in an isolated PR. Since https://github.com/bitmovin/bitmovin-player-ui/pull/656 has a lot of changes, I'd also like to avoid too many conflicts for them.

Reply:
OK that is very considerate of you.